quickstarts/microsoft.deviceupdate/deviceupdate-create-account-instance-iothub/azuredeploy.json

```diff
@@ -0,0 +1,190 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "accountName": {
+      "type": "string",
+      "defaultValue": "[take(concat('adu-quickstart-', uniqueString(newGuid())), 24)]",
+      "minLength": 3,
+      "maxLength": 24,
+      "metadata": {
+        "description": "The name of the account. More about limits see at https://aka.ms/iot-hub-device-update-limits"
+      }
+    },
+    "location": {
+      "type": "string",
+      "allowedValues": [
+        "westus2",
+        "northeurope",
+        "southeastasia"
+      ],
+      "metadata": {
+        "description": "The location of the account"
+      }
+    },
+    "instanceName": {
+      "type": "string",
+      "defaultValue": "[newGuid()]",
+      "minLength": 3,
+      "maxLength": 36,
+      "metadata": {
+        "description": "The name of the instance."
+      }
+    },
+    "iotHubName": {
+      "type": "string",
+      "defaultValue": "[take(concat('iot-hub-', uniqueString(newGuid())), 50)]",

```

Comment:
same as this one, the resource names need to be consistent so that a subsequent deployment won't keep creating more resources in the same rg

Reply:
@bmoore-msft please take a look now. `instance` is a child resource under `account` so I feel that its name matters less, account's does more.